### DISCUSSION TITLE
asking for add-on please

### DISCUSSION THREAD

**Discussion by sac23** [community/community#128239] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

hello @seerge hope all is well i was wondering can u please add on battery off battery profiles we can add so that it automatically goes into example silent mode ,eco,60hz on battery and plugged in it auto goes into turbo ultimate gpu 240 hz display i dont know if that possible now but like the Legion Toolkit just wanted to ask for this thank u guys in advance

**Comment by ebndev** [comment]
Hi @sac23,

Thanks for being a part of the GitHub Community, we're glad you're here!

The Profile category is for conversations related to your GitHub profile page. I have moved this post to General, however, it looks like you're looking for support from a specific project. You might want to try asking for help somewhere that focuses on this project, such as opening an issue in the project's repository or seeing if they have their own Discussions page.

We want to make sure you’re getting the best support you can and that your message goes where you intended, but this may not be the right place.

Best of luck!

**Reply by sac23** [reply]
Thank u I appreciate it but where can I ask this question without it getting closed ahh I see ok I'll try to see if I can do that  thanks

**Comment by ebndev** [comment]
If it's for the g-helper project, it looks like they have their own [Discussions](https://github.com/seerge/g-helper/discussions) forum!

There has been a bug lately on posts ending up in Profile (unless you ended up here through clicking on other links) but if you create a new discussion in the link above, the maintainers of that repo should see your post.

**Reply by sac23** [reply]
Ur awesome thank u so much for your help

**Reply by ebndev** [reply]
Hi again @sac23!

If you don't mind, would you mind letting us know how you initially ended up in the [Profile category](https://github.com/orgs/community/discussions/categories/profile?discussions_q=is%3Aopen+category%3AProfile+) when creating your discussion? We are trying to investigate if there is a bug or link that may be re-directing users from the project's Discussions board, if that’s where you intended to post.

Any insight would be helpful, thanks!

**Reply by ebndev** [reply]
Never mind, found the issue! Thanks again for your time and being a part of the GitHub Community 😄

**Comment by seerge** [comment]
@sac23 hello, you can already achieve everything that you have asked.

- App remembers last used mode per plugged in / on battery case and automatically switches that. So just set Silent when being on battery once and it will switch next time.
- For Display refresh rate just set ``Auto`` and it will set 60hz on battery and maximum refresh when plugged in 
- For GPU - set `Optimized` mode - it will disable dGPU (i.e. set ``Eco``) on battery and enable dGPU (i.e. set `Standard` when plugged in). You can't automatically set Ultimate as it requires reboot.

Thanks

**Reply by sac23** [reply]
u are great yup working like u said that u so much really appreciate it

**Reply by sac23** [reply]
Sorry about that the late reply I'm glad you found it thank you for all your help
